oidcclient/jwk.go

```diff
@@ -161,44 +145,6 @@ type ProviderJSON struct {
 	UserInfoURL string `json:"userinfo_endpoint"`
 }
 
-// JSONWebKeySet represents the data which is returned by the tenants /oauth2/certs endpoint
```

Comment:
not sure whether we can get rid of it... with multiple tenant support we may need to call the endpoint with extra header...

Reply:
We still have full control about the request to the endpoint. Only parsing of the result is done by the lib. As long as there are no non-standard fields in the result we should be fine.